src/components/Categories.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Categories = () => (
+  <div>
+    <h3>Coming soon...</h3>
+  </div>
```

Comment:
- [ ] According to the requirements there should only be a button with the text "Check Status" in this component.